src/steamship/data/block.py

```diff
@@ -15,13 +17,26 @@ class BlockQueryRequest(Request):
     tag_filter_query: str
 
 
+class BlockUploadType(str, Enum):
+    FILE = "file"  # A file uploaded as bytes or a string
+    BLOCKS = "blocks"  # Blocks are sent to create a file
+    URL = "url"  # content will be fetched from a URL
+    NONE = "none"  # No upload; plain text only
+
+
 class Block(CamelModel):
+    """A Block is a chunk of content within a File. It can be plain text content, image content,
+    video content, etc. If the content is not text, the text value may be the empty string
```

Comment:
If I read the create() function correctly we also support text=None which is in conflict with having to use an empty string for backwards compatibility.